examples/xml-record-conversion/xml_record_conversion.bal

```diff
@@ -0,0 +1,58 @@
+import ballerina/io;
+import ballerina/xmldata;
+
+// Defines a record type with annotations.
+@xmldata:Namespace {
+    prefix: "ns",
+    uri: "http://sdf.com"
+}
+type Invoice record {
+    string address;
+    Item[] items;
+    @xmldata:Attribute
+    string 'xmlns;
+    @xmldata:Attribute
+    string attr?;
+    @xmldata:Attribute
+    string ns\:attr?;
+};
+
+type Item record {
+    string|ItemCode itemCode;
+    int count;
+};
+
+@xmldata:Namespace {
+    uri: "http://example.com"
+}
+type ItemCode record {
+    @xmldata:Attribute
+    string discount;
+    string \#content;
```

Comment:
Don't put `\#content` here. This is a special case. we don't need to cover everything in one BBE